/**:
  ros__parameters:
    device_name: /dev/motor
    baud_rate: 9600
    flow_control: none
    parity: none
    stop_bits: "1"
